apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx

```diff
@@ -159,6 +208,23 @@ export const GlobalAppsContainer: React.FC = () => {
         }
     }, [handleMessage])
 
+    // Way to tell global iframe id of created task because it generated on condo side
+    useEffect(() => {
+        if (!isEmpty(objs) && !isEmpty(tasks) && iframeLoadedCount === iframeRefs.current.length) {
+            for (const iframe of iframeRefs.current) {
+                const targetOrigin = extractOrigin(iframe.src)
+                const targetWindow = get(iframe, 'contentWindow', null)
+
+                if (targetWindow) {
+                    sendMessage({
+                        type: TASK_PROCESSING_STATUS,
+                        data: { tasks: tasks.map(task => task.record).filter(task => task.status === TASK_STATUS.PROCESSING) },
```

Comment:
Will every iframe get all tasks including tasks, not belonging to it?

Reply:
Good point, working on it right now

Reply:
Suppose I fixed this issue by last changes made